library/src/main/java/com/novoda/downloadmanager/notifications/NotificationTag.java

```diff
@@ -0,0 +1,68 @@
+package com.novoda.downloadmanager.notifications;
+
+import android.app.Notification;
+
+import com.novoda.downloadmanager.lib.DownloadBatch;
+
+/**
+ * Build tag used for collapsing several {@link DownloadBatch} into a single
+ * {@link Notification}.
+ */
+public class NotificationTag {
```

Comment:
Generating a `toString` would make it easier to debug should anything go wrong that involves the `NotificationTag`.